docs/guides/mfa/15_totp.mdx

```diff
@@ -0,0 +1,146 @@
+---
+id: totp
+title: Google Authenticator (TOTP)
```

Comment:
```suggestion
title: TOTP (Google Authenticator)
```

@aeneasr I think I see why you've changed the title to put google authenticator first - TOTP might not say much to ppl who are new to the game, but Google Authenticator is a widely used tool. Similar to the Yubikey thing, I'd rather put the technology first, so how about we include the app name in parenthesis?